src/Microsoft.AspNetCore.Server.HttpSys/NativeInterop/ComNetOS.cs

```diff
@@ -15,11 +15,7 @@ static ComNetOS()
         {
             var win8Version = new Version(6, 2);
 
-#if NETSTANDARD1_3
-            IsWin8orLater = (new Version(RuntimeEnvironment.OperatingSystemVersion) >= win8Version);
-#else
             IsWin8orLater = (Environment.OSVersion.Version >= win8Version);
```

Comment:
The Windows check happens much earlier:
https://github.com/aspnet/HttpSysServer/blob/50ed4564f1c2fe936a43358d268b89fe7d32f018/src/Microsoft.AspNetCore.Server.HttpSys/HttpSysListener.cs#L56

Haven't actually tried it on mac/linux, I expect it to be pretty explosive.